sharq/sharq.conf

```diff
@@ -1,7 +1,7 @@
 [sharq]
-job_expire_interval       : 1000 ; in milliseconds
-job_requeue_interval      : 1000 ; in milliseconds
-default_job_requeue_limit : -1 ; retries infinitely
+job_expire_interval       : 120000 ; in milliseconds
+job_requeue_interval      : 5000 ; in milliseconds
+default_job_requeue_limit : 0 ; value of -1 retries infinitely
```

Comment:
if we set requeue_limit then all the requeue request will populate job_discard_list, I think we should set this value to 1, although this is a local configuration but you can put the same in consul-cfg repository . (voice / sms)

Reply:
This is intentionally set to 0 to give consumers (sms/voice) some time to make changes necessary. We don't want the 'fixed requeue' to introduce any regression in application logic which currently assumes that requeue is broken. As you pointed out, the goal is to eventually derive this config value from consul. Applications will have to intentionally configure a value greater than 0 to turn on requeuing.